File: python/Komponen.py
class Komponen:
    def __init__(self, nama="", merk="", tahun_rilis=0, harga=0):
        self.nama = nama
        self.merk = merk
        self.tahun_rilis = tahun_rilis
        self.harga = harga
    
    def setNama(self, nama):
        self.nama = nama
    
    def setMerk(self, merk):
        self.merk = merk
    
    def setTahunRilis(self, tahun_rilis):
        self.tahun_rilis = tahun_rilis
    
    def setHarga(self, harga):
        self.harga = harga
    
    def getNama(self):
        return self.nama
    
    def getMerk(self):
        return self.merk
    
    def getTahunRilis(self):
        return self.tahun_rilis
    
    def getHarga(self):
        return self.harga

File: cpp/Komponen.cpp
#pragma once
#include <iostream>
#include <string>
using namespace std;

class Komponen {
private:
    string nama;
    string merk;
    int tahun_rilis;
    int harga;

public:
    Komponen() {}
    
    Komponen(string nama, string merk, int tahun_rilis, int harga) {
        this->nama = nama;
        this->merk = merk;
        this->tahun_rilis = tahun_rilis;
        this->harga = harga;
    }
    
    void setNama(string nama) {
        this->nama = nama;
    }
    
    void setMerk(string merk) {
        this->merk = merk;
    }
    
    void setTahunRilis(int tahun_rilis) {
        this->tahun_rilis = tahun_rilis;
    }
    
    void setHarga(int harga) {
        this->harga = harga;
    }
    
    string getNama() {
        return nama;
    }
    
    string getMerk() {
        return merk;
    }
    
    int getTahunRilis() {
        return tahun_rilis;
    }
    
    int getHarga() {
        return harga;
    }
    
    ~Komponen() {}
};